server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -222,6 +225,12 @@ public ClientAuthConfig getClientAuthConfig() {
     return clientAuthConfig;
   }
 
+  public Long getMaximumSecretSizeInBytes() { return maximumSecretSizeInBytes; }
+
+  public void setMaximumSecretSizeInBytes(Long maximumSecretSizeInBytes) {
+    this.maximumSecretSizeInBytes = maximumSecretSizeInBytes;
+  }
```

Comment:
This property also has a setter for ease of testing (also, there's precedent for a handful of other properties). Without this you have to rely on Jackson or some other framework to transform JSON/YAML/etc. into a POJO, which feels like overkill.

Reply:
Maybe leave a comment for this setter method saying that it its main purpose is to make testing easier and not intended for use outside of testing?

Reply:
Normally I would slap on a @VisibleForTesting but I wasn't sure what the purpose of the other setters was (they aren't similarly marked). I'll go ahead and add this.